libs/permit-utils/project.json

```diff
@@ -0,0 +1,41 @@
+{
+  "name": "permit-utils",
+  "$schema": "../../node_modules/nx/schemas/project-schema.json",
+  "sourceRoot": "libs/permit-utils/src",
+  "projectType": "library",
+  "targets": {
+    "build": {
+      "executor": "@nx/vite:build",
+      "outputs": ["{options.outputPath}"],
+      "options": {
+        "outputPath": "dist/libs/permit-utils"
+      }
+    },
+    "publish": {
+      "command": "node tools/scripts/publish.mjs permit-utils {args.tag} {args.otp}",
+      "dependsOn": ["build"]
```

Comment:
Command used to publish:

```bash
npx nx run permit-utils:publish [--tag <tag>] [--otp <otp>]
```

Right now published to my personal account with a different package name https://www.npmjs.com/package/alfetopito-permit-utils?activeTab=code